main.py

```diff
@@ -120,20 +149,64 @@ def update_board(board):
     return board
 
 
+def draw_fps_slider(y, fps_limit):
+    """Draws the slider with the y coordinate of the button click
+       (How many FPS this corresponds to is not dealt with here.)"""
+    c = config.Size * config.Width + config.Edge / 2 + config.ButtonSize / 2  # The middle of the line of buttons
+    d = config.StartOfSlider
```

Comment:
c? d? e? Try to make better variable names

Reply:
done